docs/cmdline_specifying.md

```diff
@@ -117,5 +117,15 @@ Here is an extract from a Java core file that shows the options that are used:
         2CIUSERARG               -XXallowvmshutdown:true
         2CIUSERARG               -Xoptionsfile=test1.test_options_file
 
+6.  The `_JAVA_OPTIONS` environment variable. You can override previous options using this environment variable. The options that you specify with this environment variable are added to the end of the command line when a VM starts in that environment. The environment variable can contain multiple blank-delimited argument strings, but must not contain comments. For example:
+
+    - On Windows systems:
+
+            set _JAVA_OPTIONS="-Dmysysprop1=tcpip -Dmysysprop2=wait -Xdisablejavadump"
```

Comment:
I don't think the quotes are appropriate on Windows.

Reply:
Can you fix the similar issue in the section for `OPENJ9_JAVA_OPTIONS` too, please?